ui/frontend/src/components/ActivityStats.vue

```diff
@@ -19,7 +20,7 @@
         <h3>Summary:</h3>
 
         <ul>
-          <li v-for="activity in activities">
+          <li v-for="(activity, index) in activities" :key="index">
```

Comment:
teach me! what does `:key="index"` do?

Reply:
When you do a `v-for`, if you don't provide a `key`, Vue doesnt know which element corresponds to which array item. So Vue has to perform a re-render on all elements even if only 1 element in the array was mutated. This creates that mapping so Vue only re-renders the element that changed 💨

Reply:
cool!